src/Templates/Components/modals/modal-forgot.pug

```diff
@@ -0,0 +1,111 @@
+script.
+  document.addEventListener('DOMContentLoaded', function() {
+    var identityInput = document.getElementById('identity');
+    var agencyIdInput = document.getElementById('agencyId');
+    if (agencyIdInput) {
+      agencyIdInput.addEventListener('invalid', function(e) {
+        e.target.setCustomValidity("Feltet skal udfyldes");
+      });
+
+      agencyIdInput.addEventListener('input', function(e) {
+        e.target.setCustomValidity("");
+      });
+      
+      agencyIdInput.addEventListener('input', function (e) {
+        let value = e.target.value;
+        // Trim to 6 characters
+        value = value.substring(0, 6);
+        e.target.value = value;
```

Comment:
Could this not be done in a one-liner like: 
e.target.value = e.target.value.substring(0, 6);

Reply:
done